eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -193,7 +193,9 @@ function Set-ChangeLogContent {
 
   try
   {
-    $ChangeLogEntries = $ChangeLogEntries.Values | Sort-Object -Descending -Property ReleaseStatus, ReleaseVersion
+    $ChangeLogEntries = $ChangeLogEntries.Values | Sort-Object -Property `
```

Comment:
This is the wrong order. We need to sort by date first, then version. This would be correct:

```powershell
$ChangeLogEntries = $ChangeLogEntries.Values | Sort-Object -Property -Descending -Property `
  ReleaseStatus, `
  @{e = {[AzureEngSemanticVersion]::new($_.ReleaseVersion)}}
```

Reply:
I have updated this. I looked at the tests and they work correctly. The only problem was that it was passed as a `string` rather than `AzureEngSemanticVersion`